ui/raidboss/data/05-shb/alliance/the_copied_factory.js

```diff
@@ -874,23 +759,20 @@
         'Marx [LR]': '(Linker|Rechter) Marx',
         'Medium Exploder': 'mittelgroße Selbstzerstörung',
         'Multi-leg Medium Model': 'mittelgroßes mehrbeiniges Modell',
-        'Quality assurance will be sealed off': 'Noch 15 Sekunden, bis sich die Warenkontrollhalle schließt',
+        'Quality assurance': 'Warenkontrollhalle',
```

Comment:
Thanks! I thought I fixed these, but maybe I missed them.

Reply:
No this was fixed by you, but I did something wrong, git went wrong somehow.

Reply:
This branch is forked from maybe two weeks ago? so those changes are not adopted yet here.